### ISSUE TITLE
Should Observer and Host classes be equivalent to or superclass of Sensor and Platform?

### ISSUE THREAD

**Issue by alexrobin** [w3c/sdw#1408]
In PR #1402, `Observer` and `Host` are super classes of their `Sensor` and `Platform` counterparts. They are also kept more abstract because they are not `Systems`.

**Comment by alexrobin** [comment]
@KathiSchleidt FYI, all of these classes (`Observer`, `Host`, `Sensor`, `Platform`) are already in the SOSA namespace. `Deployment` is the one that is not.

Please see my other [comment](https://github.com/w3c/sdw/issues/1411#issuecomment-1540831487) regarding making `Platform` a `System`.

**Comment by alexrobin** [comment]
@KathiSchleidt You are right. I go confused because the `usedProcedure` association between `Observation` and `Procedure` is defined in SOSA but the `implements` association between `Sensor` and `Procedure` actually comes from SSN (inherited from System). The interleaving of the two namespaces is definitely confusing...

So I see what you mean now... If `Observer` is not a subclass of System, it doesn't get the association to `Procedure` automatically. However, the way I defined it (see [here](https://github.com/w3c/sdw/pull/1402/files#diff-bddd5642ff86775ffb333c45b693b0b76f4d23fbe90e3b20f4082c5dda6a8134R229)), it still gets the `implements` property whether or not it inherits from System. Also note that the range of `implements` was not limited to `System` in the existing ontology.

So I guess things can still be made to work even if `Observer` is kept independent from `System` but it is definitely a point of discussion.

**Comment by KathiSchleidt** [comment]
@alexrobin Turning this discussion around, what is your rational for keeping `Observer` independent from `System`?

In addition, whatever we decide for `Platform`, the same should apply for `Host`
